website/docusaurus.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
     footer: {
       logo: {
         alt: 'moja global light logo',
-        src: 'static/img/logo-light.png',
+        src: './static/img/logo-light.png',
```

Comment:
@sarthakkundra I guess changing `src: './static/img/logo-light.png',` to `src: 'img/logo-light.png',` might work!
Reference: https://docusaurus.io/docs/docusaurus.config.js/#optional-fields

![Screenshot from 2021-07-17 01-40-33](https://user-images.githubusercontent.com/56754747/126003799-1f1b22b9-a734-43ff-8bc6-9b6b9d3ee410.png)

Reply:
Thanks for the reference. Trying this now.